src/ui/actions/timeline.ts

```diff
@@ -179,37 +163,18 @@ export async function getInitialPausePoint(recordingId: string) {
     return { point, time };
   }
 
-  const firstMeaningfulPaint = await getFirstMeaningfulPaint();
+  const firstMeaningfulPaint = await findFirstMeaningfulPaint();
   if (firstMeaningfulPaint) {
     const { point, time } = firstMeaningfulPaint;
     return { point, time };
   }
 }
 
 export function setHoverTime(time: number | null, updateGraphics = true): UIThunkAction {
-  return async (dispatch, getState) => {
-    dispatch(setTimelineState({ hoverTime: time }));
-
-    if (!updateGraphics) {
-      return;
-    }
-
-    const stateBeforeScreenshot = getState();
-
-    try {
-      const currentTime = getCurrentTime(stateBeforeScreenshot);
-      const screenshotTime = time || currentTime;
-      const { screen, mouse } = await getGraphicsAtTime(screenshotTime);
-      const stateAfterScreenshot = getState();
-
-      if (getHoverTime(stateAfterScreenshot) !== time) {
-        return;
-      }
-
-      paintGraphics(screen, mouse);
-    } catch (error) {
-      console.error(error);
-    }
+  return dispatch => {
```

Comment:
:hand: If this is no longer a thunk, then the entire body of `setHoverTime` could be replaced with just `return setTimeLineState(args)` (ie, it's just a semantic wrapper for the `setTimelineState` action)

Reply:
Sure 👍🏼